tests/scripts/generate_tls13_compat_tests.py

```diff
@@ -487,17 +648,20 @@ def get_all_test_cases():
                               CIPHER_SUITE_IANA_VALUE.keys(),
                               NAMED_GROUP_IANA_VALUE.keys(),
                               SIG_ALG_IANA_VALUE.keys()):
-            yield generate_compat_test(client=client, server=server,
-                                       cipher=cipher, named_group=named_group,
-                                       sig_alg=sig_alg)
+            if server == 'mbedTLS' or client == 'mbedTLS':
+                yield generate_compat_test(client=client, server=server,
+                                           cipher=cipher, named_group=named_group,
+                                           sig_alg=sig_alg)
+
 
         # Generate Hello Retry Request  compat test cases
         for client, server, client_named_group, server_named_group in \
             itertools.product(CLIENT_CLASSES.keys(),
                               SERVER_CLASSES.keys(),
                               NAMED_GROUP_IANA_VALUE.keys(),
                               NAMED_GROUP_IANA_VALUE.keys()):
-            if client_named_group != server_named_group:
+            if (client == 'mbedTLS' or server == 'mbedTLS') and \
```

Comment:
More like python code.
```suggestion
            if 'mbedTLS' in (client, server) and \
```